tests/views/test_game.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from mars_api.views import GameViewSet
+
+GAME_PATH = "mars_api/games/"
+
+pytestmark = pytest.mark.django_db
+
+
+def test_can_post_game_with_player_scores():
+    """Test if a game can be created alongside PlayerScores in one POST."""
```

Comment:
Purely from a logical perspective, should this not be the other way around?

Reply:
One cannot have a score for a `Game` that does not exist? While a `Game` can exist, but maybe it wasn't finished (no scores) . :thinking:

Reply:
It's just about wording